Android/quickstart-kotlin/app/build.gradle

```diff
@@ -0,0 +1,38 @@
+apply plugin: 'com.android.application'
+
+apply plugin: 'kotlin-android'
+
+apply plugin: 'kotlin-android-extensions'
+
+android {
+    compileSdkVersion 28
+    defaultConfig {
+        applicationId "com.example.immersivereadersdk"
+        minSdkVersion 15
+        targetSdkVersion 28
+        versionCode 1
+        versionName "1.0"

```

Comment:
what is this version for? if it's the android sdk version should it be the same as the js version it targets?

Reply:
This is bootstrapped into the app that supports all Android versions by default. I would not recommend changing anything here.